meta.json

```diff
@@ -1,13 +1,13 @@
 {
-  "module_id": "viam-labs:viam-appliedmotion",
+  "module_id": "viam:viam-appliedmotion",
   "visibility": "public",
-  "url": "https://github.com/viam-labs/viam-appliedmotion",
+  "url": "https://github.com/viam-modules/viam-appliedmotion",
   "description": "A modular component for Applied Motion Motor Controllers",
   "models": [
     {
       "api": "rdk:component:motor",
-      "model": "viam-labs:appliedmotion:st"
+      "model": "viam:appliedmotion:st"
     }
   ],
   "entrypoint": "viam-appliedmotion"
-}
\ No newline at end of file
+}
```

Comment:
Missing newline I think?

Reply:
the file does have a newline idk why it's not showing up here

Reply:
empty lines won't show in github diffs at the end of a file, it'll only show you the red circle with horizontal line if there is no newline at the end of the file.